man/org.freedesktop.systemd1.xml

```diff
@@ -4852,6 +4856,14 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       <varname>StateDirectory</varname>, <varname>CacheDirectory</varname> and <varname>LogsDirectory</varname>,
       which will create a symlink of the given name to the respective directory. The messages take an unused
       <varname>flags</varname> parameter, reserved for future backward-compatible changes.</para>
+
+      <para><varname>ExtraFileDescriptorNames</varname> contains file descriptor names passed to the service via
+      the ExtraFileDescriptors property in the StartTransient dbus API. The file descriptor names are passed to
+      the service manager via
+      <citerefentry><refentrytitle>sd_notify</refentrytitle><manvolnum>3</manvolnum></citerefentry>.
+      While ExtraFileDescriptors contains the file descriptors and their names, ExtraFileDescriptorNames only
+      contains names. File descriptor number may change when sent to dbus and thus showing the number in
+      ExtraFileDescriptorNames may introduce confusion.</para>
```

Comment:
`<varname>` here too at various places.

I'd drop the last sentence here. It's not really so much about the renumbering. Properties that carry fds are really weird, because you inspect an object and bam it would install an fd into your fd space. that would be very weird. Hence we shouldn't do it. But this discussion doesn't have to be mentioned here, it's not the right place. Hence just drop the last senetnce.